mandatory/3-function-output.js

```diff
@@ -11,6 +11,7 @@ function combine2Words(word1, word2) {
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  return firstWord.concat(" ", secondWord, " ", thirdWord);
```

Comment:
Please provide **comments** to these functions before these functions. Comments help programmer to understand the usage of the function and facilitate IDE auto suggestions / error detection.